lib/web_ui/test/engine/window_test.dart

```diff
@@ -407,6 +407,145 @@ Future<void> testMain() async {
     js_util.setProperty(domWindow, 'screen', original);
   });
 
+  // Emulates the framework sending a request to get screen orientation.
+  Future<List<dynamic>> sendGetPreferredOrientations() {
+    final Completer<List<dynamic>> completer = Completer<List<dynamic>>();
+    final ByteData? inputData = const JSONMethodCodec().encodeMethodCall(
+        const MethodCall('SystemChrome.getPreferredOrientations'));
+
+    window.sendPlatformMessage(
+      'flutter/platform',
+      inputData,
+      (ByteData? outputData) {
+        const MethodCodec codec = JSONMethodCodec();
+        completer.complete(codec.decodeEnvelope(outputData!) as List<dynamic>);
+      },
+    );
+
+    return completer.future;
+  }
+
+  test('getPreferredOrientation responds', () async {
+    final DomScreen original = domWindow.screen!;
+
+    final List<String> lockCalls = <String>[];
```

Comment:
This test doesn't seem to use this variable for anything meaningful. It just puts stuff in it then clears it.